src/bootstrap.js

```diff
@@ -4,8 +4,21 @@
 
 /* eslint-disable no-unused-vars */
 
+Components.utils.import("resource://gre/modules/Services.jsm");
+
 function startup({webExtension}, reason) {
+  Services.telemetry.registerEvents("lockbox", {
```

Comment:
I wonder if this should go in global scope. We might want to have events for enabling/disabling the add-on, and `startup` is only run when the add-on is enabled.